Tests/OpenPassTests/TelemetryEventTests.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  TelemetryEventTests.swift
+//
+// MIT License
+//
+// Copyright (c) 2025 The Trade Desk (https://www.thetradedesk.com/)
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+//
+
+import Foundation
+import InlineSnapshotTesting
+@testable import OpenPass
+import XCTest
+
+final class TelemetryEventTests: XCTestCase {
+    
+    func testEventTelemetryInfo() {
+        let event = TelemetryEvent(
+            clientId: "client-id-13",
+            name: "An event",
+            message: "of telemetry",
+            eventType: .info
+        )
+        assertInlineSnapshot(of: event, as: .json) {
+            """
+            {
+              "clientId" : "client-id-13",
```

Comment:
The JSON keys are of a different format to Android, `clientId` vs `client_id`. That shouldn't be expected, right?